arcade/experimental/clock/clock_window.py

```diff
@@ -777,16 +780,18 @@ def show_view(self, new_view: AnyView):
         """
         if not isinstance(new_view, View):
             raise TypeError(
-                f"Window.show_view() takes an arcade.View,"
-                f"but it got a {type(new_view)}.")
+                f"Window.show_view() takes an arcade.View," f"but it got a {type(new_view)}."
```

Comment:
If we have one line, then no need for two separate f-strings. This should be merged into one.
```suggestion
                f"Window.show_view() takes an arcade.View, but it got a {type(new_view)}."
```